source/includes/api/_subscriptions.md

```diff
@@ -1,704 +1,811 @@
 # Subscriptions
 
-Subscription represents to which datasets and analysis a user can have access to.
+## What is a subscription?
 
-<aside class="notice">
-Remember — All subscription endpoints need to be authenticated.
-</aside>
+A subscription allows you to get notified of updates on a datasets' data. We strongly recommend that you read the [dataset concept](#dataset) and [dataset endpoints](#dataset6) sections before proceeding.
```

Comment:
Could be useful here to include a couple of examples of why you might want subscriptions.